source/features/anonymize-tweets.js

```diff
@@ -0,0 +1,7 @@
+export default function () {
+	$('.FullNameGroup .fullname').text('Jane Doe');
+	$('.js-retweet-text b').text('Jane Doe');
```

Comment:
Same as https://github.com/sindresorhus/refined-twitter/pull/173/files#r267718771

Reply:
How about `Math.random() >= 0.5 ? 'Jane Doe' : 'John Doe'`? That way we stay totally neutral.

Reply:
For sure, I changed it